src/components/split-button/split-button-classic.style.js

```diff
@@ -0,0 +1,75 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+import StyledButton from '../button/button.style';
+
+const classicColors = {
+  active: '#1963f6',
+  primary: '#255bc7',
+  secondary: '#1e499f',
+  tertiary: '#163777'
+};
+
+const getSplitButtonChildrenClassicStyles = ({ theme }) => (theme.name === THEMES.classic ? css`
```

Comment:
if I'll use `&&` operator, when the conditional results in false then a `"false"` string will be returned, which is not a valid CSS. It will be ignored by the browser and displayed correctly, but it could lead to an untraceable error during snapshot generation.